releases/2017_cycle_3.md

```diff
@@ -25,9 +25,12 @@ Scope:
     - More facility level metadata
     - Removal of `NAN`s from metadata
 - `Tlist` Shutter Protocol
```

Comment:
@tacaswell has said that he would welcome a PR into bluesky with a `per_step` for count.